app/src/main/java/com/ucmobiledevelopment/freeways/RetrofitClientInstance.kt

```diff
@@ -7,7 +7,7 @@ object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
     private val BASE_URL = "https://crashviewer.nhtsa.dot.gov/"
 
-    val retrofitInstance : Retrofit?
+    val retrofitInstance: Retrofit?
```

Comment:
Deleted space according to Kotlin convention. https://kotlinlang.org/docs/coding-conventions.html#colon